<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>IdentityEditorWin</class>
 <widget class="QDialog" name="IdentityEditorWin">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>373</width>
    <height>252</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Identity editor</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <property name="leftMargin">
    <number>0</number>
   </property>
   <property name="topMargin">
    <number>0</number>
   </property>
   <property name="rightMargin">
    <number>0</number>
   </property>
   <property name="bottomMargin">
    <number>0</number>
   </property>
   <item>
    <layout class="QFormLayout" name="formLayout">
     <item row="0" column="0">
      <widget class="QLabel" name="lNick">
       <property name="text">
        <string>Nick</string>
       </property>
      </widget>
     </item>
     <item row="1" column="0">
      <widget class="QLabel" name="lIdent">
       <property name="text">
        <string>Ident</string>
       </property>
      </widget>
     </item>
     <item row="2" column="0">
      <widget class="QLabel" name="lRealName">
       <property name="text">
        <string>Real name</string>
       </property>
      </widget>
     </item>
     <item row="3" column="0">
      <widget class="QLabel" name="lAway">
       <property name="text">
        <string>Away message</string>
       </property>
      </widget>
     </item>
     <item row="0" column="1">
      <widget class="QLineEdit" name="le_Nick"/>
     </item>
     <item row="1" column="1">
      <widget class="QLineEdit" name="le_Ident"/>
     </item>
     <item row="2" column="1">
      <widget class="QLineEdit" name="le_RealName"/>
     </item>
     <item row="3" column="1">
      <widget class="QLineEdit" name="le_AwayMsg"/>
     </item>
    </layout>
   </item>
   <item>
    <spacer name="verticalSpacer">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>81</height>
      </size>
     </property>
    </spacer>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QPushButton" name="pbSave">
       <property name="text">
        <string>Save</string>
       </property>
       <property name="default">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="pbCancel">
       <property name="text">
        <string>Cancel</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
